### ISSUE TITLE
NX659J R lacks dts and does not have the arch/arm64/boot/dts/vendor directory

### ISSUE THREAD

**Issue by toss-a** [ztemt/NX659J-R#2]
Lack of dts, hope the Nubian team can supplement the missing dts

**Comment by mrslezak** [comment]
Yes, impossible to build this kernel, someone please 7z or zip up the vendor directory so we can build it.  The Q version had qcom, nubia, and bindings as subdirectories.  This one lacks all 3.

**Comment by mrslezak** [comment]
I mean seriously you guys uploaded a 7zip split archive all you had to do is include the vendor directory and it would be fine.  I know Nubia has been clear that they don't support devs, this is a gaming phone, I get it.  But at least when you upload code UPLOAD ALL OF IT PLEASE.  Many of us like to tinker with the settings to get the gameplay / functionality up on the phone and this is just free labor to attract others to your brand.

**Comment by Ack-Draper** [comment]
Is this feasible? Extract the device tree file (DTB) from my device, then decompile it into a device tree source file (DTS) to fill in the missing files in the arch/arm64/boot/dts/vendor directory.
